src/main/java/io/ebeaninternal/server/persist/platform/OracleHelpImpl.txt

```diff
@@ -0,0 +1,26 @@
+package io.ebeaninternal.server.persist.platform;
+
+import java.sql.Array;
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import oracle.jdbc.OracleConnection;
```

Comment:
NOTE: this is a text file, as oracle.jdbc.OracleConnection is not public available. So I included the class in binary form:
https://github.com/ebean-orm/ebean/pull/1328/files#diff-1d0fda0b6639c184544f2e59c5ae5036

Do you think this hack is acceptable?